scripts/gefs_atmos_f0306.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/ksh
```

Comment:
This script should start like this. A brief description of this script should be added as well.  

```suggestion
#! /usr/bin/env bash

source "${USHgfs}/preamble.sh"
```